frontend/test/resolution/testModuleInitOrder.cpp

```diff
@@ -40,9 +40,151 @@ static std::string chplHome() {
   return ret;
 }
 
+static void
+checkMentionedModules(Context* ctx, ID idMod, ...) {
```

Comment:
Given that this is a test and memory use is not critical, consider just accepting a `std::vector<const char*>`, and passing the names by wrapping them in curly braces. This would help avoiding the C-style varargs API. Also, I suspect that if `findMentionedModules` reported more modules than the test expects, you'll go straight into reading undefined memory. A vector (using `at(idx)`) would provide bounds checking.

Reply:
I did consider that, but I opted instead to follow along with the style that this test was already using, as IMO if we change this one we should also change checkModuleInitOrder. I can do that though.